chanacceptor/errors.go

```diff
@@ -5,15 +5,32 @@ import (
 	"fmt"
 )
 
+const (
+	// Length of the prefix message in `ErrRejectedWithMsg` + the max. length of
+	// custom message allowed to be sent to the remote peer.
+	trimmedAcceptanceErrorLength = 31 + 128
```

Comment:
This commit should probably be in the commit that introduces the AcceptanceError. Otherwise the diff gets larger b/t commits because of test changes.